cli_tools/gce_vm_image_import/importer/api_inflater.go

```diff
@@ -43,19 +43,24 @@ func isCausedByAlphaAPIAccess(err error) bool {
 
 // apiInflater implements `importer.inflater` using the Compute Engine API
 type apiInflater struct {
-	serialLogs    []string
-	args          ImportArguments
-	computeClient daisyCompute.Client
-	storageClient storage.Client
+	serialLogs      []string
+	args            ImportArguments
+	computeClient   daisyCompute.Client
+	storageClient   storage.Client
+	guestOsFeatures []*computeAlpha.GuestOsFeature
 }
 
 func createAPIInflater(args ImportArguments, computeClient daisyCompute.Client, storageClient storage.Client) inflater {
-	return &apiInflater{
+	inflater := apiInflater{
 		serialLogs:    []string{},
 		args:          args,
 		computeClient: computeClient,
 		storageClient: storageClient,
 	}
+	if args.UefiCompatible {
```

Comment:
Why not just do it inside apiInflater.inflate()? In my opinion, this is too detail and not worth show up in high-level logic.

Reply:
I chose this location since it was the most unit-testable place that I could find.

Since you're actively working in this file, do you see a good place that's also testable?